components/Editor.js

```diff
@@ -284,7 +284,7 @@ class Editor extends React.Component {
     return (
       <div className="editor">
         <Toolbar>
-          <Themes key={theme} updateTheme={this.updateTheme} theme={theme} />
+          <Themes key={theme} update={this.updateState} theme={theme} highlights={highlights} />
```

Comment:
@raboid I don't think `key` is necessary anymore, and in fact it will hurt rendering. Is this correct?

Reply:
Yeah good catch 👍